lib/house.rb

```diff
@@ -0,0 +1,65 @@
+class House
+  attr_reader :address, :rooms
+
+  def initialize(price_paramater, address_parameter)
+    @price = price_paramater
+    @address = address_parameter
+    @rooms = []
+  end
+
+  def price
+    @price.gsub("$", '').to_i
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def above_market_average?
+    price > 500000 ? true : false
```

Comment:
This is fine, although you don't need the ternary here at all. 

```
def above_market_average?
   price > 500000
end
```

`price > 500000` is a comparison and thus returns the correct boolean all by itself, so no need for an explicit true/false